src/FSAL/FSAL_CORTXFS/conf/nfs_setup.sh

```diff
@@ -303,6 +297,65 @@ function cortx_nfs_config {
 	echo -e "\nNFS setup is complete"
 }
 
+function cortx_ganesha_restart {
+	echo -e "\nStopping NFS Ganesha"
+	systemctl stop nfs-ganesha || die "Failed to stop NFS-Ganesha"
+
+	echo -e "\nUpdating conf file"
+        cat > $GANESHA_CONF << EOM
+# An example of KVSFS NFS Export
+EXPORT {
+
+        # Export Id (mandatory, each EXPORT must have a unique Export_Id)
+        Export_Id = 12345;
```

Comment:
Can we have a fixed export ID for dummy export ? So that, the dummy export ID would not clash with actual one.

Reply:
This is the exact dummy export block that we use when running setup script.